src/Hazelcast.Net.Examples/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
pretty sure we don't want to include that file - if you keep adding it by accident maybe add a rule to `.gitignore`?

Reply:
You are right. It keeps coming because of consecutive relation between linq PRs. I'll definitely add it to gitignore.

Reply:
Fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/732/commits/cea4111d7f8995ee8ea7fad26026d61c33d9d206